collect_app/src/main/res/drawable/ic_add_location_black.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="36.000001dp"
+        android:height="36.000001dp"
+        android:viewportWidth="36.000001"
+        android:viewportHeight="36.000001">
```

Comment:
Could it be just 36 in 4 lines above? I think that 000001 doesn't change anything.